### PR TITLE
Find boost::filesystem explicitly

### PR DESCRIPTION
**Pull request by galexv**
...because ALPSCore does not provide it anymore.

### CODE REVIEW

File: cmake/FindGSL.cmake [hunk 1]
```diff
@@ -91,8 +91,7 @@ ELSE(WIN32)
       "${GSL_ROOT_DIR}/bin"
       CACHE STRING "preferred path to GSL (gsl-config)")
     FIND_PROGRAM(GSL_CONFIG gsl-config
-      ${GSL_CONFIG_PREFER_PATH}
-      /usr/bin/
```
Comment (galexv) [inline cmake/FindGSL.cmake]: This change is unintentional. Disregard it if you feel that "/usr/bin" must be listed explicitly.

**Comment by ryanlevy** [comment]
I'm confused, I thought I removed all `boost::filesystem` dependence. Is it lurking somewhere?

**Comment by galexv** [comment]
> I thought I removed all boost::filesystem dependence. Is it lurking somewhere?

For example, here: https://github.com/CQMP/Maxent/blob/0defb322e297e08f6a62027be05953bfc55d7c82/src/maxent.cpp#L23

But if it is the only occurrence, feel free to reject the PR in favor of removing the references to boost FS.

**Comment by ryanlevy** [comment]
I see, its been properly removed in the GPLv3 branch, but not in the master. @egull how do you think we should proceed?

**Comment by egull** [comment]
Remove it from the master. I see no reason to have it there if it's gone in GPLv3

**Pull request closed without merging**